pandas/tests/indexes/test_multi.py

```diff
@@ -632,6 +632,13 @@ def test_from_arrays_index_series_period(self):
 
         tm.assert_index_equal(result, result2)
 
+    def test_from_arrays_different_lengths(self):
+        # GH13599
+        idx1 = [1, 2, 3]
```

Comment:
I am not sure if I can assert explicitly that the ValueError isn't raised.

Should I simply add an example where array lengths are same and assert the response is the MultiIndex object with valid data?